packages/suite/src/views/wallet/sign-verify/index.tsx

```diff
@@ -74,155 +76,190 @@ interface FormEvent {
-                                name="verifyAddress"
-                                value={verifyAddress}
-                                onChange={this.handleInputChange}
-                                type="text"
-                                state={verifyAddressError ? 'error' : undefined}
-                                bottomText={verifyAddressError ? verifyAddressError.message : null}
-                            />
-                        </Row>
-                        <Row>
-                            <Textarea
-                                topLabel={intl.formatMessage(messages.TR_MESSAGE)}
-                                name="verifyMessage"
-                                value={verifyMessage}
-                                onChange={this.handleInputChange}
-                                rows={4}
-                                maxRows={4}
-                                maxLength={255}
-                            />
-                        </Row>
-                        <Row>
-                            <Textarea
-                                topLabel={intl.formatMessage(messages.TR_SIGNATURE)}
-                                name="verifySignature"
-                                value={verifySignature}
-                                onChange={this.handleInputChange}
-                                rows={4}
-                                maxRows={4}
-                                maxLength={255}
-                            />
-                        </Row>
-                        <RowButtons>
-                            <StyledButton onClick={signVerifyActions.clearVerify}>
-                                <Translation id="TR_CLEAR" />
-                            </StyledButton>
-                            <StyledButton
-                                // isDisabled={!!verifyAddressError || !device.connected}
-                                isDisabled
-                                onClick={() => {
-                                    if (errors.length <= 0) {
-                                        signVerifyActions.verify(
-                                            verifyAddress,
-                                            verifyMessage,
-                                            verifySignature,
-                                        );
-                                    }
-                                }}
-                            >
-                                <Translation id="TR_VERIFY" />
-                            </StyledButton>
-                        </RowButtons>
-                    </Verify>
-                </Wrapper>
-            </WalletLayout>
-        );
-    }
-}
+    const handleInputChange = (event: FormEvent) => {
+        signVerifyActions.inputChange(event.target.name as InputNameType[0], event.target.value);
+    };
+
+    const getAddress = (): AddressToSign => {
```

Comment:
`AddressToSign` is unnecessary, this method will always return object { address, path } therefore additional typings is not required

Reply:
you confused me, I thought we were writing in typescript (nor javascript), and you say that sometimes typing is not necessary (?); when it is necessary, and when it is not necessary in your subjective opinion, no one explained to me